community/contributing/index.rst

```diff
@@ -11,6 +11,7 @@ Contributing
    bisecting_regressions
    code_style_guidelines
    bug_triage_guidelines
+   content_guidelines
    documentation_guidelines
    docs_writing_guidelines
    updating_the_class_reference
```

Comment:
I think we might need subsections in Contributing as we start having a number of guidelines specific to documentation contributions, and it's not obvious from their title which covers what and in what order they should be read (specifically the "content", "documentation" and "docs writing" guidelines).

Reply:
Well that's exactly what #4067 does, so all good :)